components/proxy/src/main/java/com/hotels/styx/proxy/plugin/InstrumentedPlugin.java

```diff
@@ -42,21 +43,23 @@
     private static final Logger LOGGER = getLogger(InstrumentedPlugin.class);
 
     private final NamedPlugin plugin;
-    private final SimpleCache<HttpResponseStatus, Meter> errorStatusMetrics;
-    private final SimpleCache<Class<? extends Throwable>, Meter> exceptionMetrics;
-    private final Meter errors;
+    private final SimpleCache<HttpResponseStatus, Counter> errorStatusMetrics;
+    private final SimpleCache<Class<? extends Throwable>, Counter> exceptionMetrics;
+    private final Counter errors;
 
     public InstrumentedPlugin(NamedPlugin plugin, Environment environment) {
         this.plugin = requireNonNull(plugin);
         requireNonNull(environment);
 
+        Metrics.addRegistry(environment.meterRegistry());
+
         this.errorStatusMetrics = new SimpleCache<>(statusCode ->
-                environment.metricRegistry().meter("plugins." + plugin.name() + ".response.status." + statusCode.code()));
+                Metrics.counter("plugins." + plugin.name() + ".response.status." + statusCode.code()));
 
         this.exceptionMetrics = new SimpleCache<>(type ->
-                environment.metricRegistry().meter("plugins." + plugin.name() + ".exception." + formattedExceptionName(type)));
+                Metrics.counter("plugins." + plugin.name() + ".exception." + formattedExceptionName(type)));
```

Comment:
Similarly here, plugin name and exception class could be tags, leaving the meter being called "plugin.exception"